src/node/miner.cpp

```diff
@@ -158,10 +158,13 @@ std::unique_ptr<CBlockTemplate> BlockAssembler::CreateNewBlock()
     CMutableTransaction coinbaseTx;
     coinbaseTx.vin.resize(1);
     coinbaseTx.vin[0].prevout.SetNull();
+    coinbaseTx.vin[0].nSequence = CTxIn::MAX_SEQUENCE_NONFINAL;
```

Comment:
Note to reviewers: `IsFinal()` has been present since the first commit as a way to ignore the sequence number, hence the need to avoid `SEQUENCE_FINAL`.

The proposed consensus rule doesn't require `MAX_SEQUENCE_NONFINAL`, anything other than `SEQUENCE_FINAL` is allowed. Maybe that's good to say in a comment.

Reply:
Added a comment.